torchdata/datapipes/iter/load/huggingface.py

```diff
@@ -17,42 +17,48 @@
 
 
 def _get_response_from_huggingface_hub(
-    dataset: str, split: str, revision: str, streaming: bool, data_files: Optional[Dict[str, str]]
+    path: str, streaming=True, split="train", revision="main", **config_kwargs
 ) -> Iterator[Any]:
-    hf_dataset = datasets.load_dataset(
-        dataset, split=split, revision=revision, streaming=streaming, data_files=data_files
-    )
+    hf_dataset = datasets.load_dataset(path=path, streaming=streaming, split=split, revision=revision, **config_kwargs)
     return iter(hf_dataset)
 
 
 class HuggingFaceHubReaderIterDataPipe(IterDataPipe[Tuple[str, StreamWrapper]]):
     r"""
-    Takes in dataset names and returns an Iterable HuggingFace dataset
-    Args format is the same as https://huggingface.co/docs/datasets/loading
+    Takes in dataset names and returns an Iterable HuggingFace dataset.
+    Args format and meaning are the same as https://huggingface.co/docs/datasets/loading.
+    Contrary to their implementation, default behavior differs in the following:
+    split is set to "train".
+    revision is set to "main".
+    streaming is set to True.
```

Comment:
nit: Add a tab before each line
Add one empty line before `Args`

Reply:
Would it be better to align this with the default arguments of the load_dataset() function (e.g. split is None)? 
Or would this lead to problems elsewhere? A quick Github search reveals no usage of the HuggingFaceHubReader outside of Pytorch Data.

Reply:
Good question.
For PyTorch project, we want to have a deprecation cycle for those kind of BC-breaking changes. You can keep `split` as `train` for now and add a deprecation warning to indicate users that `train` will be changed to `None` in the release `0.7`